doc/command_ref.rst

```diff
@@ -480,7 +485,10 @@ Dumps lists of packages depending on the packages' relation to the
 system. A package is ``installed`` if it is present in the RPMDB, and it is ``available``
 if it is not installed but it is present in a repository that DNF knows about.
 The list command can also limit the displayed packages according to other criteria,
-e.g. to only those that update an installed package.
+e.g. to only those that update an installed package. The exclude option in
```

Comment:
can you put there a ref for exclude option too, please? Then you can push it to the master.